speed-dreams/source-2.2.3/src/libs/robottools/rthumandriver.cpp

```diff
@@ -59,10 +59,18 @@
 #include <playerpref.h>
 #include <car.h>
 
+
 #include "humandriver.h"
 #if SDL_FORCEFEEDBACK
 #include "forcefeedback.h"
 
+// ASSISTED DRIVING ASSISTANCE: added recorder
+// To record uncomment the #define RECORD_SESSION 1 in backend/ConfigEnums.h
+#include <Recorder.h>
+#include <ConfigEnums.h>
+Recorder* recorder;
```

Comment:
should this also be surrounded with a #if RECORD_SESSION guard?

Reply:
yes that could be done